src/System.Private.CoreLib/shared/System/Runtime/Intrinsics/Arm/Base.cs

```diff
@@ -0,0 +1,60 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Runtime.CompilerServices;
+
+namespace System.Runtime.Intrinsics.Arm
+{
+    /// <summary>
+    /// This class provides access to the ARM base hardware instructions via intrinsics
+    /// </summary>
+    [Intrinsic]
+    [CLSCompliant(false)]
+    public abstract class Base
```

Comment:
Btw, didn't you decide to rename it to something less generic (ArmBase?)

Reply:
Yes, I just confirmed with @terrajobst. Will fix to `ArmBase` and update the types to inherit from it as appropriate.